src/ring_jwt_middleware/core.clj

```diff
@@ -284,22 +284,25 @@
                 (handle-error (format "(%s) %s"
                                       (or (jwt->user-id jwt) "Unknown User ID")
                                       (str/join ", " validation-errors))
-                              {:jwt jwt
+                              {:request request
```

Comment:
Looks like the addition of the `request` can mean that it can end up being logged (see `defn default-error-handler`).

Is there a chance that we could end up logging sensitive info?

Reply:
Yes good catch you should probably `dissoc` the `request` from the `pr-str` of the `default-error-handler`.